mmdet/engine/runner/loops.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+from mmengine.model import is_model_wrapper
+from mmengine.runner import ValLoop
+
+from mmdet.registry import LOOPS
+
+
+@LOOPS.register_module()
+class MultiValLoop(ValLoop):
```

Comment:
This Loop requires the model has teacher and student. So it is not general enough to use this name. Maybe rename to `TeacherStudentValLoop`.